### DISCUSSION TITLE
steam mass message to friends feature

### DISCUSSION THREAD

**Discussion by ultninjaF** [3urobeat/steam-mass-comment-bot#25] [Ideas]
It would be nice to have such a feature in the bot like a mass sending of messages to friends.

**Comment by ultninjaF** [comment]
The bot name could be "steam-mass-comment-and-message-bot "
:D

**Comment by 3urobeat** [comment]
I'm not exactly sure if I'd like to implement that. It would make spamming users quite a lot more easy and could be abused easily.  
Commenting on profiles has at least a greater amount of rate limiting and content checking.
